atlas/src/views/SearchView/VideosResultTab.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Search_search_item_Video } from '@/api/queries/__generated__/Search'
+
+type VideosResultTabProps = {
+  videos: Search_search_item_Video[]
+  loading: boolean
+}
+
+const VideosResultTab: React.FC<VideosResultTabProps> = () => {
+  return <div>VideosResultTab</div>
```

Comment:
I think this component might actually have been left here?

Reply:
Yep, removed